test/python/transpiler/test_vf2_layout.py

```diff
@@ -119,6 +119,28 @@ def test_call_limit(self):
             pass_.property_set["VF2Layout_stop_reason"], VF2LayoutStopReason.NO_SOLUTION_FOUND
         )
 
+    def test_coupling_map_and_target(self):
+        """Test that a Target is used instead of a CouplingMap if both are specified."""
+        cmap = CouplingMap([[0, 1], [1, 2], [2, 0]])
+        target = Target()
+        target.add_instruction(CXGate(), {(0, 1): None, (1, 2): None, (1, 0): None})
+        qr = QuantumRegister(3, "qr")
+        circuit = QuantumCircuit(qr)
+        circuit.cx(qr[0], qr[1])  # qr0-> qr1
+        circuit.cx(qr[1], qr[2])  # qr1-> qr2
```

Comment:
Does this test actually do what it says?  It looks to me like the circuit is valid under both coupling maps, whereas I'd have expected the `CouplingMap` argument to be impossible to satisfy, if we wanted to test the `target`?

Reply:
That's a good point. I probably meant to have a `cx qr[1] qr[0]` here and forgot to add it. I'll fix this up now

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7735/commits/7ffec344defda4f47d73f759a4079995bfc61ea1